src/coloring/acyclic_coloring.jl

```diff
@@ -10,29 +10,30 @@ is a collection of trees—and hence is acyclic.
 Reference: Gebremedhin AH, Manne F, Pothen A. **New Acyclic and Star Coloring Algorithms with Application to Computing Hessians**
 """
 function color_graph(g::LightGraphs.AbstractGraph, ::AcyclicColoring)
-
     color = zeros(Int, nv(g))
-    forbidden_colors = zeros(Int, nv(g))
+    set = DisjointSets{Int}([])
```

Comment:
Give `set` a more informative name

```suggestion
    set = DisjointSets{Int}(())
```